packages/shared/src/components/Feed.tsx

```diff
@@ -137,6 +140,12 @@ export default function Feed<T>({
   const numCards = currentSettings.numCards[spaciness ?? 'eco'];
   const isSquadFeed = feedName === 'squad';
   const { shouldUseFeedLayoutV1 } = useFeedLayout();
+  const showAcquisitionForm =
+    isLoggedIn &&
```

Comment:
Anyone on myFeed is already logged in :)

Reply:
Good to know, we can remove that check then 👍